optuna/_convert_positional_args.py

```diff
@@ -0,0 +1,65 @@
+from functools import wraps
+from inspect import signature
+from typing import Any
+from typing import Callable
+from typing import Sequence
+from typing import TypeVar
+import warnings
+
+from typing_extensions import ParamSpec
+
+
+_T = TypeVar("_T")
+_P = ParamSpec("_P")
+
+
+def convert_positional_args(
+    *,
+    previous_positional_arg_names: Sequence[str],
+    warning_stacklevel: int = 2,
+) -> Callable[[Callable[_P, _T]], Callable[_P, _T]]:
+    """Convert positional arguments to keyword arguments
```

Comment:
Sorry but a conventional nit.
```suggestion
    """Convert positional arguments to keyword arguments.
```